src/cart/swim/swim_internal.h

```diff
@@ -50,8 +50,8 @@ extern "C" {
 /** SWIM protocol parameter defaults */
 #define SWIM_PROTOCOL_PERIOD_LEN 1000	/* milliseconds */
 #define SWIM_SUSPECT_TIMEOUT	(8 * SWIM_PROTOCOL_PERIOD_LEN)
-#define SWIM_PING_TIMEOUT	900	/* milliseconds */
-#define SWIM_SUBGROUP_SIZE	2
+#define SWIM_PING_TIMEOUT	1800	/* milliseconds */
+#define SWIM_SUBGROUP_SIZE	3
```

Comment:
Why you need to increase this? It will bring more RPCs communication. Do we really need this?

Reply:
just to reduce possible false positive, in DAOS-8221's log, there are lots of SWIM RPC timeout and some ranks marked as dead "swim_member_dead() member x is DEAD".

quoted from LANL paper:
Protocol accuracy is particularly poor at k values of 1 or 2 for high message loss rates. With a 5% message loss rate almost all servers are falsely confirmed as failed. With a subgroup size of 3 there is only a single false positive at a message loss rate of 5%, although we still observe that nearly all servers are suspected at some point. At a subgroup size of 4, the number of suspect servers declines and the protocol no longer produces any false positives at any message loss rate. With subgroup sizes of 5 or 6 the number of suspected servers diminishes because of the decreased likelihood of all direct and indirect pings failing for a given target.